src/features/content-answers/ContentAnswersOpenButton.scss

```
@@ -0,0 +1,20 @@
+@import '../../styles/variables';
+
+.bdl-ContentAnswersOpenButton {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    width: 40px;
+    height: 40px;
+    background-color: $bdl-gray-02;
+    border: none;
+
+    .btn-content {
+        display: flex;
+    }
+
+    &:focus,
+    &:not(.bdl-is-disabled):hover {
```

sorry missed this but we'll likely want the `is-disabled` check for focus too
```suggestion
    &:not(.bdl-is-disabled):focus,
    &:not(.bdl-is-disabled):hover {
```